guard/src/rules/exprs.rs

```diff
@@ -364,15 +364,13 @@ impl<'loc> std::fmt::Display for AccessQuery<'loc> {
 
 impl<'loc> std::fmt::Display for FunctionExpr<'loc> {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}(", self.name)?;
-        let last_index = self.parameters.len() - 1;
-        for (idx, each_param) in self.parameters.iter().enumerate() {
-            write!(f, "{}", each_param)?;
-            if idx != last_index {
-                write!(f, ", ")?;
-            }
-        }
-        write!(f, ")")?;
+        let params = self
+            .parameters
+            .iter()
+            .map(|each| each.to_string())
+            .collect::<Vec<_>>()
+            .join(",");
+        write!(f, "({})", params)?;
         Ok(())
```

Comment:
nit: i think we can remove the `?` at the end of the `write!` macro, and also remove the `Ok(())` return since the [return from the macro is the same as this function ](https://doc.rust-lang.org/std/macro.write.html)

Reply:
Tight ty